src/seedcase_sprout/core/__init__.py

```diff
@@ -88,9 +89,10 @@
     # "delete_resource_data",
     # "delete_resource_properties",
     # Path -----
-    "path_package",
-    "path_properties",
-    "path_packages",
+    path_package,
+    path_packages,
+    path_properties,
+    path_readme,
```

Comment:
```suggestion
    "path_package",
    "path_packages",
    "path_properties",
    "path_readme",
```

They need to be in quotes.